forge-gui/res/cardsfolder/a/ardenn_intrepid_archaeologist.txt

```diff
@@ -2,11 +2,8 @@ Name:Ardenn, Intrepid Archaeologist
 ManaCost:2 W
 Types:Legendary Creature Kor Scout
 PT:2/2
-T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigTarget | TriggerZones$ Battlefield | OptionalDecider$ You | TriggerDescription$ At the beginning of combat on your turn, you may attach any number of Auras and Equipment you control to target permanent or player.
-SVar:TrigTarget:DB$ Pump | ValidTgts$ Permanent,Player | TgtPrompt$ Select target permanent or player | RememberObjects$ Targeted | SubAbility$ DBAttach
-SVar:DBAttach:DB$ RepeatEach | UseImprinted$ True | RepeatSubAbility$ DBRepeatAttach | SubAbility$ DBCleanup | RepeatCards$ Aura.YouCtrl,Equipment.YouCtrl
-SVar:DBRepeatAttach:DB$ Attach | Defined$ Remembered | Object$ Imprinted | Optional$ True
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigAttach | TriggerZones$ Battlefield | OptionalDecider$ You | TriggerDescription$ At the beginning of combat on your turn, you may attach any number of Auras and Equipment you control to target permanent or player.
+SVar:TrigAttach:DB$ Attach | Defined$ Targeted | ValidTgts$ Permanent,Player | TgtPrompt$ Select target permanent or player | Object$ Valid Aura.YouCtrl,Equipment.YouCtrl | Optional$ True
```

Comment:
i checked AttachEffect again, while it works, we should probably Enhance AttachEffect

* when giving list of `Object$` and `Optional$ True` and maybe `Defined$` is used (it does fall back to Targeted anyway when using `ValidTgts`)  Filter and Pick the Aura and Equipments via Multiselect, and then skip the later optional check.
* with that it would be A faster for Player to select which ones to attach to target
* You don't get asked for stuff you can't equip to target like Equipment to a Player

but that can be optimized in later MR?